docs/4.3/admin-guide.md

```diff
@@ -2259,11 +2261,58 @@ When upgrading multiple clusters:
 
 ## Backing Up Teleport
 
+When planning a backup of Teleport, it's important to know what is where and the
+importance of each component. Teleports Proxies and Nodes are stateless, and thus
+only `teleport.yaml` should be backed up.
+
+The Auth server is Teleports brains, and depending on the backend should be backup
+in a regular cadance.
+
+For example a customer running Teleport on AWS with etd have these key items of data:
+
+| What | Where ( Example AWS Customer ) |
+|-|-|
+| Local Users ( not SSO )  | DynamoDB |
+| Certificate Authorities | DynamoDB |
+| Trusted Clusters | DynamoDB |
+| Connectors: SSO | DynamoDB / File System  |
+| RBAC | DynamoDB / File System |
+| teleport.yaml | File System |
+| teleport.service | File System |
+| license.pem | File System |
+| TLS /. | ( File System / Outside Scope )  |
+
+For this customer, we would Recommend using [AWS best practices](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/BackupRestore.html) for backing up DynamoDB. If DynamoDB is used for
+the audit log, logged events have a TTL of 1 year.
+
+| Backend | Recommend Backup Strategy  |
+|-|-|
+| dir ( local filesystem )   | Copy `data_dir/storage` and use `tctl get all` to get local state. |
```

Comment:
I'd rephrase the strategy as: "Backup `/var/lib/teleport/storage` directory and the output of `tctl get all`".
Also, if a user backs up `/var/lib/teleport`, is there any reason to do `tctl get all`?

Reply:
I think the idea with `tctl get all` was that the format would be YAML and therefore backend agnostic, so theoretically you could dump out the CAs, users etc and then restore these into a different backend.